engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,22 +231,44 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float function1(float parameter){
```

Comment:
See: https://github.com/MovingBlocks/Terasology/wiki/Common-Style-Points#descriptive-variable-names

Reply:
Indeed, please use a _descriptive name_ here. What is this function doing? What to the numbers mean?